config/toml.go

```diff
@@ -547,26 +547,24 @@ max_open_connections = {{ .Instrumentation.MaxOpenConnections }}
 # Instrumentation namespace
 namespace = "{{ .Instrumentation.Namespace }}"
 
-# The URL of the influxdb instance to use for remote event
-# collection. If empty, remote event collection is disabled.
-influx_url = "{{ .Instrumentation.InfluxURL }}"
+# The URL of the trace db to use for remote event
+# collection. If empty, remote event pushing is disabled.
+trace_push_url = "{{ .Instrumentation.TracePushURL }}"
 
-# The influxdb token to use for remote event collection.
-influx_token = "{{ .Instrumentation.InfluxToken }}"
+# The tracer url token to use for remote event collection.
+# If empty, remote event pulling is disabled.
+trace_pull_address = "{{ .Instrumentation.TracePullAddress }}"
 
-# The influxdb bucket to use for remote event collection.
-influx_bucket = "{{ .Instrumentation.InfluxBucket }}"
-
-# The influxdb org to use for event remote collection.
-influx_org = "{{ .Instrumentation.InfluxOrg }}"
+# The tracer to use for collecting trace data.
+trace_type = "{{ .Instrumentation.TraceType }}"
 
 # The size of the batches that are sent to the database.
-influx_batch_size = {{ .Instrumentation.InfluxBatchSize }}
+trace_push_batch_size = {{ .Instrumentation.TraceBufferSize }}
 
 # The list of tables that are updated when tracing. All available tables and
 # their schema can be found in the pkg/trace/schema package. It is represented as a
 # comma separate string. For example: "consensus_round_state,mempool_tx".
-influx_tables = "{{ .Instrumentation.InfluxTables }}"
+tracing_tables = "{{ .Instrumentation.TracingTables }}"
```

Comment:
I actually don't think this is breaking in the sense that it will require operators to make any modifications. They will be able to continue running their node as normal.

It is breaking if you had previously been using the influx tracing but I don't think anyone is. If we wanted to be more careful about this we should mention this as an experimental feature